corehq/apps/hqmedia/templates/hqmedia/manage_paths.html

```diff
@@ -23,10 +23,19 @@ <h3 class="panel-title">{% trans "Download Paths" %}</h3>
                 {% endblocktrans %}
             </div>
 
-            <a class="btn btn-primary" href="{% url 'download_multimedia_paths' domain app_id %}">
-                <i class="fa fa-cloud-download"></i>
-                {% trans "Download Paths" %}
-            </a>
+            <form class="form-horizontal" method="GET" action="{% url 'download_multimedia_paths' domain app_id %}">
+                <div class="checkbox">
+                    <label>
+                        <input type="checkbox" name="only_missing" />
+                        {% trans "Include only paths where the multimedia file is missing" %}
```

Comment:
Does missing mean that the path is present in all media i.e its present somewhere in the app but its not present in the final mapping on the app that is used to provide phone with the multimedia?
Just want to confirm my understanding because "Has not been provided" and "Missing" are messing my interpretation.

Reply:
Yes: it means there's a reference in the app, but there's no corresponding entry in the app's mapping (presumably because no file was ever uploaded).